pyproject.toml

```diff
@@ -12,8 +12,8 @@ exclude = ["main"]
 keywords = ["django","migration","rollback"]
 
 [tool.poetry.dependencies]
-python = ">=3.6,<4.0"
-Django = "^3.2.6"
+python = ">=3.7,<4.0"
```

Comment:
The latest versions of `wheel` do not support py36, only py37+, so I had to change this.